src/utils/gamepad.ts

```diff
@@ -1,7 +1,8 @@
-﻿import EventEmitter from 'eventemitter3';
+﻿import Save from '@dreamirl/dreamengine/src/utils/Save';
+import EventEmitter from 'eventemitter3';
 import config from '../config';
 import Events from './Events';
-import InputsManager, { InputType, Inputs } from './Inputs';
+import Inputs, { InputType } from './Inputs';
```

Comment:
Need to keep previous naming to prevent compatibility with old project for update